### ISSUE TITLE
admin: Customer VAT-ID only visible in "edit" but not in "show"

### ISSUE THREAD

**Issue by laf0rge** [shopware/platform#2634]
### PHP Version

7.4.30

### Shopware Version

6.4.12.0

### Expected behaviour

The VAT ID[s] of the Customer should be visible in the admin interface when looking at the customer detail view https://shop.foo.bar/admin#/sw/customer/detail/...

### Actual behaviour

The VAT ID[s] are not show in the detail page.  Only once you edit th customer, suddenly you see an Input box with the VAT ID.

This is quite consfusing as while viewing the customer it looks like no VAT-ID was entered by the customer, but in reality it is just not shown by shopware.

### How to reproduce

Plain 6.4.12.0 installation. Simulate a user making an order while registering a user account with EU-VAT ID during checkout.

Then log into the admin interface and look at the customer/detail page.  It doesn't state the VAT-ID the customer has entered anywhere.

**Comment by MarcelSchmaeing** [comment]
Hello,

Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

If you feel like this issue is still impacting you, please create a new issue and let us know.
Thank you again for your valuable feedback.
